dataland-document-manager/src/test/kotlin/org/dataland/documentmanager/services/conversion/XlsxToExcelConverterTest.kt

```diff
@@ -0,0 +1,49 @@
+package org.dataland.documentmanager.services.conversion
+
+import org.apache.tika.Tika
+import org.dataland.datalandbackendutils.exceptions.InvalidInputApiException
+import org.dataland.datalandbackendutils.utils.sha256
+import org.dataland.documentmanager.services.TestUtils
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.springframework.mock.web.MockMultipartFile
+
+class XlsxToExcelConverterTest {
+    private val correlationId = "test-correlation-id"
+    private val testXlsx = "sampleFiles/sample.xlsx"
+    private val emptyXlsx = "sampleFiles/EmptyExcelFile.xlsx"
+    private val xlsxToExcelConverter = XlsxToExcelConverter(mockClamAvClient())
+    private val testFileName = "test.xlsx"
+    private val mimeType = "application/x-tika-ooxml"
+
+    @Test
+    fun `verify that a xlsx file can be converted to xlsx`() {
```

Comment:
maybe indicate that the conversion only has the function of a validation in this context